Cargo.toml

```diff
@@ -8,42 +8,42 @@ description = "High level networking for the Bevy game engine"
 readme = "README.md"
 repository = "https://github.com/lifescape-game/bevy_replicon"
 keywords = [
-  "bevy",
-  "multiplayer",
-  "netcode",
-  "replication",
-  "server-authoritative",
+    "bevy",
+    "multiplayer",
+    "netcode",
+    "replication",
+    "server-authoritative",
 ]
 categories = ["game-development", "network-programming"]
 license = "MIT OR Apache-2.0"
 include = ["/benches", "/src", "/tests", "/LICENSE*"]
 
 [dependencies]
-bevy_renet = "0.0.9"
-bevy = { version = "0.11", default-features = false, features = ["bevy_scene"] }
+bevy_renet = { git = "https://github.com/lucaspoffo/renet", branch = "master" }
```

Comment:
Only until renet releases a new version

Reply:
Could you disable default features for renet since we no longer depend on specific transport after this release? And add it to `dev-dependencies` to include `transport` that used for unit tests.

Reply:
done

Reply:
Ah nvm, transport is still needed for `NetcodeClientPlugin`. Will add back the default features

Reply:
Right, we need to drop `NetcodeClientPlugin` and `NetcodeServerPlugin` initialization. This way users will be able to use any transport they want.

Reply:
Makes sense! What do you think about doing that in a separate PR as it's not strictly necessary for this update?

Reply:
Okay!

Reply:
In the current master - yes.